client/blocks/signup-form/social.jsx

```diff
@@ -50,11 +50,15 @@ class SocialSignupForm extends Component {
 			return;
 		}
 
+		this.props.recordTracksEvent( 'calypso_signup_social_button_success', {
+			social_account_type: 'google',
+		} );
+
 		this.props.handleResponse( 'google', tokens.access_token, tokens.id_token );
 	};
 
 	trackSocialLogin = ( service ) => {
```

Comment:
Should this be renamed for consistency? Now that we are trying to differentiate signup and login events, reading `trackSocialLogin` and `calypso_signup_*` event being recorded in it threw me off.

```suggestion
	trackSocialSignup = ( service ) => {
```

Reply:
> Should this be renamed for consistency?

@vindl Yep, great catch. Renamed in 352c301744